project.clj

```diff
@@ -81,9 +81,17 @@
                  [ring/ring-json "0.4.0"]                             ; Ring middleware for reading/writing JSON automatically
                  [stencil "0.5.0"]                                    ; Mustache templates for Clojure
                  [toucan "1.0.3"                                      ; Model layer, hydration, and DB utilities
-                  :exclusions [honeysql]]]
+                  :exclusions [honeysql]]
+
+                 ; Notebook utilities and prototypes
+                 [bigml/histogram "4.1.3"]
+                 [bigml/sketchy "0.4.1"]
+                 [net.cgrand/xforms "0.9.2"]
+                 [kixi/stats "0.3.8"]
+                 [redux "0.1.4"]]
```

Comment:
if these are only used for dev notebooks (i.e. the don't need to be included with the built uberjar) we should make them `:dev` dependencies instead